src/lib.rs

```diff
@@ -904,22 +904,20 @@ pub trait WebauthnConfig {
     /// If the attestation statement attStmt successfully verified but is not trustworthy per step
     /// 16 above, the Relying Party SHOULD fail the registration ceremony.
     ///
-    /// The default implementation of this method rejects None and Uncertain attestation, and
+    /// The default implementation of this method rejects ECDAA and Uncertain attestation, and
```

Comment:
I'd make it just "rejects Uncertain" because when we add support for ECDAA we'll "not" blindly reject it.